crates/proof-of-sql/benches/README.md

```diff
@@ -8,9 +8,10 @@ To run benchmarks with Jaeger, you need to do the following
     ```bash
     docker run --rm -d --name jaeger -p 6831:6831/udp -p 16686:16686 jaegertracing/all-in-one:latest
     ```
-2. Run the benchmark.
+2. Run a benchmark.
     ```bash
-    cargo bench -p proof-of-sql --bench jaeger_benches
+    cargo bench -p proof-of-sql --bench jaeger_benches InnerProductProof
+    cargo bench -p proof-of-sql --bench jaeger_benches Dory --features="test"
```

Comment:
Why the discrepancy?

Reply:
Dory's public parameters are generated with `proof_of_sql::proof_primitive::dory::test_rng()`. This function is defined in the `rand_util` module which is only added with the `--features="test"` flag.

Perhaps there is a more optimal way to do this?

Reply:
@jacobtrombetta I wonder whether we can remove https://github.com/spaceandtimelabs/sxt-proof-of-sql/blob/main/crates/proof-of-sql/src/proof_primitive/dory/mod.rs#L30 though. Unless there is some good reason why the entire module is behind that flag.

Reply:
Good idea, I'll check with @JayWhite2357 to see if he has any hesitations removing that flag.